go/database/mpt/io/archive.go

```diff
@@ -100,6 +100,9 @@ func ExportArchive(directory string, out io.Writer) error {
 
 	// Encode diff of each individual block.
 	for block := uint64(0); block <= maxBlock; block++ {
+		if interrupt(ctx) {
+			return errors.Join(archive.Close(), errors.New("failed exporting content: export interrupted"))
```

Comment:
should we consider an interrupted export as an error state?

Reply:
Thinking about it now its probably a better not to return an error.